buildSrc/build.gradle.kts

```diff
@@ -6,8 +6,7 @@ repositories {
 dependencies {
     gradleApi()
     implementation("org.jetbrains.kotlin.jvm", "org.jetbrains.kotlin.jvm.gradle.plugin", "1.9.0")
```

Comment:
Should we use 1.9.10 to match the version [here](https://github.com/JetBrains/Exposed/blob/4b5b04a154120de07a2dd7951ae19041cc932378/settings.gradle.kts#L18)?

Reply:
Fixed